eng/pipelines/libraries/enterprise/linux.yml

```diff
@@ -49,7 +50,8 @@ steps:
   displayName: Test linuxclient connection to web server
 
 - bash: |
-    docker exec linuxclient bash /repo/libraries.sh
+    docker exec linuxclient bash /repo/src/coreclr/build.sh -release -skiptests -clang9
```

Comment:
@safern should we be hardcoding release here? shouldn't this be a parameter comming form a Matrix here?

Reply:
This particular pipeline is for testing the enterprise scenarios. It should always just use the -release CoreCLR.

Reply:
Got it.

Reply:
@davidsh why `/src/coreclr/build.sh` and not `/coreclr.sh` ?

Reply:
Because some of the scripts are still broken due to the 'live-live' changes. And using the coreclr/build.sh script was the only way to get things working efficiently.